Week3/assignment/transaction.js

```diff
@@ -0,0 +1,97 @@
+import {createNewConnection, useDatabase} from "./connection_query.js";
+
+const connection = createNewConnection();
+
+const transferAmount = (fromAccount, toAccount, amount) => {
```

Comment:
Few things to consider:
- Don't you want to check if the `fromAccount` actually contains the money available for the transfer?
- What if they account does not even exist? Do you not want to just rollback then instead of storing it in `account_changes`?

Bonus:
- You can wrap query functions that you use here to use promises, that way you  can use `async-awaits` and simplifies the code a lot.